content/en/service_catalog/scorecards/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Service Scorecards
+aliases:
+  - /tracing/service_catalog/scorecards
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Service Catalog"
+- link: /api/latest/service-scorecards/
+  tag: "Documentation" 
+  text: "Service Scorecards API" 
+- link: "https://www.datadoghq.com/blog/service-scorecards/"
+  tag: "Blog"
+  text: "Prioritize and promote service observability best practices with Service Scorecards"
+- link: "https://www.datadoghq.com/blog/datadog-custom-scorecards/"
+  tag: "Blog"
+  text: "Formalize best practices with custom Scorecards"
+- link: "/continuous_integration/dora_metrics/"
+  tag: "Documentation"
+  text: "Track DORA Metrics with Datadog" 
+---
+
+{{< callout url="#" btn_hidden="true" header="false" >}}
+Service Scorecards are in beta.
+{{< /callout >}}
```

Comment:
Should we include this beta banner in the three child pages: `custom_rules`, `scorecard_configuration`, and `using_scorecards` as well?